src/main/java/Duke.java

```diff
@@ -1,10 +1,130 @@
+import java.util.ArrayList;
+import java.util.Scanner;
 public class Duke {
-    public static void main(String[] args) {
+
+    //fields
+    private static ArrayList<Task> tasks = new ArrayList<Task>();
+    private static int counter = 0;
+
+    //utility methods
+    public static void printWithDivider(String stringWithinDivider) {
+        String breakLine = "\t____________________________________________________________";
+        System.out.println(breakLine);
+        stringWithinDivider = stringWithinDivider.replace("\n", "\n\t");
+        System.out.println("\t" + stringWithinDivider);
+        System.out.println(breakLine);
+    }
+    public static ArrayList<String> processInput(String line, String delimiter) {
+        ArrayList<String> words = new ArrayList<>();
+        int divider = line.indexOf(delimiter);
+
+        words.add(line);
+
+        if (divider != -1) {
+            words.set(0, line.substring(0, divider));
+            words.add(line.substring(divider+1));
+        }
+        return words;
+    }
+
+    //command methods
+    public static void hello() {
```

Comment:
I like how you abstracted the hello message (and other tasks), that makes your main method clean and clear :D